exchanges/okx/okx_websocket.go

```diff
@@ -1245,13 +1252,21 @@ func (ok *Okx) wsProcessTrades(data []byte) error {
 				CurrencyPair: pair,
 				Exchange:     ok.Name,
 				Side:         response.Data[i].Side,
-				Timestamp:    response.Data[i].Timestamp.Time(),
+				Timestamp:    response.Data[i].Timestamp.Time().UTC(),
 				TID:          response.Data[i].TradeID,
 				Price:        response.Data[i].Price.Float64(),
 			})
 		}
 	}
-	return trade.AddTradesToBuffer(trades...)
+	if tradeFeed {
+		for i := range trades {
+			ok.Websocket.DataHandler <- trades[i]
```

Comment:
I'm wary of the direction to send individual trades through the single channel along with everything else, but I'll test it when the trade revamp comes up, so no action required